demos/reduced-motion/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8" />
   <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
-  <title>2025 Esri Developer & Technology Summit - Reduced Motion</title>
+  <title>Reduced motion</title>
```

Comment:
nitpick:

Since the switch toggle pertains to animation and not the light/dark modes, should line 14 be omitted?